pkg/util/paramtable/service_param.go

```diff
@@ -61,16 +63,23 @@ func (p *ServiceParam) init() {
 	p.MetaStoreCfg.Init(&p.BaseTable)
 	p.EtcdCfg.Init(&p.BaseTable)
 	p.DBCfg.Init(&p.BaseTable)
+	p.MQCfg.Init(&p.BaseTable)
 	p.PulsarCfg.Init(&p.BaseTable)
 	p.KafkaCfg.Init(&p.BaseTable)
 	p.RocksmqCfg.Init(&p.BaseTable)
+	p.NatsmqCfg.Init(&p.BaseTable)
 	p.MinioCfg.Init(&p.BaseTable)
 }
 
 func (p *ServiceParam) RocksmqEnable() bool {
 	return p.RocksmqCfg.Path.GetValue() != ""
 }
 
+// NatsmqEnable checks if NATS messaging queue is enabled.
+func (p *ServiceParam) NatsmqEnable() bool {
+	return p.NatsmqCfg.Server.StoreDir.GetValue() != ""
```

Comment:
Would it not be more appropriate to include a Boolean field - 'enabled' - to signify the activation of Natsmq?